website/src/components/Analytics/index.tsx

```diff
@@ -5,7 +5,7 @@ import LinkedInInsights from "./LinkedInInsights";
 export default function Analytics() {
   return (
     <>
-      <Mixpanel />
+      {/* <Mixpanel /> */}
```

Comment:
Was this intentional? 
```suggestion
      <Mixpanel />
```